R/CAMPP2.R

```diff
@@ -74,15 +74,15 @@ runCampp2 <- function (data1, metadata1, data2=NULL, metadata2=NULL, technology,
     print("CREATING SUMMARIZED EXPERIMENT")
 
     print("Creating SE from 1st dataset.")
-    rownames(campp2_brca_1_meta)<-campp2_brca_1_meta$ID
-    campp2_brca_se_1 <- CreateSE(campp2_brca_1, campp2_brca_1_meta)
+    rownames(metadata1)<-metadata1$group1
```

Comment:
are `group1` (and `group2` for `metadata2`) guaranteed to exist in the input data frame?

Reply:
fixed

Reply:
it's checked in the `parseArguments()`